pages/ko/lb3/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 lang: ko
-title: "LoopBack 3.0 - 한국어"
+title: "LoopBack 3.0"
```

Comment:
I think it's important to keep the name of the language in this title, since this is the home page for Korean LB docs. It will make it immediately clear what the page is if (for example) someone lands there from a search.
You'll notice this is the convention followed by all other languages, and I'd like to be consistent.